code/modules/vehicles/mecha/combat/gygax.dm

```diff
@@ -98,9 +98,23 @@
 	. = ..()
 	max_ammo()
 
-/obj/vehicle/sealed/mecha/gygax/dark/add_cell(obj/item/stock_parts/cell/C=null)
+/obj/vehicle/sealed/mecha/gygax/dark/add_cell(obj/item/stock_parts/cell/C)
```

Comment:
```suggestion
/obj/vehicle/sealed/mecha/gygax/dark/add_cell(obj/item/stock_parts/cell/power_cell)
```
(sorry to lecture)
One of the ongoing efforts in the codebase is not only adding to, but improving previous code. Accessibility is huge, so changing variables to be more descriptive is a must (even if its not "your" code)

This applies to the other changes you've made as well, please address these

https://github.com/tgstation/tgstation/blob/master/.github/guides/STYLE.md#use-descriptive-and-obvious-names

Reply:
Done